pkg/util/cluster/cluster.go

```diff
@@ -140,8 +141,11 @@ func New(log *logrus.Entry, environment env.Core, ci bool) (*Cluster, error) {
 }
 
 func (c *Cluster) Create(ctx context.Context, vnetResourceGroup, clusterName string) error {
-	_, err := c.openshiftclustersv20200430.Get(ctx, vnetResourceGroup, clusterName)
+	clusterGet, err := c.openshiftclustersv20200430.Get(ctx, vnetResourceGroup, clusterName)
 	if err == nil {
+		if clusterGet.ProvisioningState == mgmtredhatopenshift20200430.ProvisioningState(api.ProvisioningStateFailed) {
+			return fmt.Errorf("cluster is in failed provisioning state, please delete and retry")
```

Comment:
I think the return message can also be - 
`return fmt.Errorf("cluster exists and is in failed provisioning state, please delete and retry")`